db/migrate/20230706194017_update_exercises.rb

```diff
@@ -0,0 +1,9 @@
+class UpdateExercises < ActiveRecord::Migration[7.0]
+  def change
+    change_table :exercises do |t|
+      t.float :weight, null: false
+      t.integer :reps, null: false
+      t.belongs_to :user, foreign_key: true
+    end
```

Comment:
We should talk about how you update an existing DB instead of always relying on a seed file. Not hard to add into this ☝️